packages/compass-editor/src/codemirror/query-history-autocompleter.ts

```diff
@@ -92,3 +108,17 @@ function createInfo(query: SavedQuery): {
     },
   };
 }
+
+// scales a number unscaledNum between [minAllowed, maxAllowed]
+export function scaleBetween(
+  unscaledNum: number,
+  minAllowed: number,
+  maxAllowed: number,
+  min: number,
+  max: number
+): number {
+  if (max === min) return 0;
```

Comment:
this looks like unexpected input. returning a number could make such input go unnoticed, and 0 might be out of the allowed scale - so it would be safer to throw

Reply:
I returned 0 because in the context of the autocompleter, originalScaleMin = originalScaleMax only when there is 1 lastExecuted time, in which case it doesn't matter how the autocompleter ranks the completion. The `boost` property in the options expects numbers between -99 and 99, so I could probably return any number between those. 

For the sake of making the input noticed, do you think it would be a better idea to throw an error in the function and then catch the error within the autocompleter options itself instead of handling it in the function?